core/modules/layout/layout.admin.inc

```diff
@@ -94,7 +94,34 @@ function layout_list_page() {
     array('data' => t('Operations'), 'class' => array('layout-operations')),
   );
 
+  $help_items = array();
+  $help_items[] = t('To determine which layout is to be used for a page, the path is compared to those shown here.');
+  $help_items[] = t('Paths used by the layout system are <em>system paths</em> (also called <em>normal paths</em>), not !link.', array('!link' => l(t('URL aliases'), 'admin/config/urls/path')));
+  $help_items[] = t('They are listed alphabetically below. All layouts that might apply to each path are listed below the path.');
+  $help_items[] = t('All pages without a layout explicitly defined for its path will use one of the <em>Default layouts</em> (shown at the bottom of this page).');
```

Comment:
- "pages" here is plural, but we later say "its path" (singular).
- Should the word "Default" be capitalized here? I think not.

Reply:
I don't think we even need that part...  `All pages without a layout explicitly defined will use one of the ...` would be fine here.